Src/Support/GoogleApis/Apis/[Media]/Upload/ResumableUpload.cs

```diff
@@ -412,7 +391,7 @@ public Task<IUploadProgress> UploadAsync()
 
             try
             {
-                UploadUri = await InitializeUpload(cancellationToken).ConfigureAwait(false);
+                UploadUri = await InitiateSessionAsync(cancellationToken).ConfigureAwait(false);

```

Comment:
Yes, consistency would be good. Let's still call the protected method InitiateSessionAsync, but call the result UploadUri, and the factory method CreateFromUploadUri.